lib/perl/Genome/VariantReporting/Suite/BamReadcount/Annotate.pm

```diff
@@ -5,16 +5,52 @@ use warnings FATAL => 'all';
 
 class Genome::VariantReporting::Suite::BamReadcount::Annotate {
     is => 'Genome::VariantReporting::Framework::Component::Expert::Command',
+    attributes_have => {
+        is_argument => {
```

Comment:
Since this is now an Expert::Command we shouldn't need to define this attributes_have